cadquery/occ_impl/shapes.py

```diff
@@ -1564,29 +1577,11 @@ def sortWiresByBuildOrder(wireList, result=[]):
     if len(wireList) < 2:
         return [wireList, ]
 
-    # make a Face
-    face = Face.makeFromWires(wireList[0], wireList[1:])
-
-    # use FixOrientation
-    outer_inner_map = TopTools_DataMapOfShapeListOfShape()
-    sf = ShapeFix_Face(face.wrapped)  # fix wire orientation
-    sf.FixOrientation(outer_inner_map)
-
-    # Iterate through the Inner:Outer Mapping
-    all_wires = face.Wires()
-    result = {w: outer_inner_map.Find(
-        w.wrapped) for w in all_wires if outer_inner_map.IsBound(w.wrapped)}
-
-    # construct the result
-    rv = []
-    for k, v in result.items():
-        tmp = [k, ]
-
-        iterator = TopTools_ListIteratorOfListOfShape(v)
-        while iterator.More():
-            tmp.append(Wire(iterator.Value()))
-            iterator.Next()
-
-        rv.append(tmp)
+    # make a Face, NB: this might return
```

Comment:
Was this supposed to say something like "might return null"?

Reply:
Indeed, fixed in the last commits. Should read: `# make a Face, NB: this might return a compound of faces`